app/src/test/java/com/github/onlynotesswent/ui/navigation/NavigationActionsTest.kt

```diff
@@ -99,4 +101,62 @@ class NavigationActionsTest {
     val overviewId = navigationActions.popFromScreenNavigationStack()
     assertNull(overviewId)
   }
+
+  @Test
+  fun testNavigateToFolderContentsWithRootFolder() {
```

Comment:
Some modularization could be added for these tests, but better to leave this for a future pr, regrouping other test modularization.